### DISCUSSION TITLE
Has POT incorporated the entropy-regularized Wasserstein loss for multilabel classification tasks?

### DISCUSSION THREAD

**Discussion by BarretBa** [PythonOT/POT#557] [Q&A]
The paper 'Learning with a Wasserstein Loss' introduces the entropy-regularized Wasserstein loss for multilabel classification tasks, but I couldn't find how to call this function. Is this functionality integrated into POT?

**Comment by rflamary** [comment]
the multilabel classification loss is actually entropic unbalanced Ot thta can be computed with:

https://pythonot.github.io/all.html#ot.sinkhorn_unbalanced2
